packages/ui/src/components/va-time-input/VaTimeInput.vue

```diff
@@ -101,14 +102,28 @@ export default defineComponent({
     const changePeriodToPm = () => changePeriod(true)
     const changePeriodToAm = () => changePeriod(false)
 
+    const reset = (): void => {
+      modelValueSync.value = undefined
+    }
+
+    const input = ref<InstanceType<typeof VaInput>>()
+
+    const focus = (): void => {
+      // we will replace '_focus' with 'focus' when will resolve 'withConfigTransport' problem with 'expose' in 'setup' func
+      input.value?._focus()
+    }
+
+    const blur = (): void => {
```

Comment:
These methods are not exposed because of withTransportConfig bug. You should use here the same approach with Options API methods.

Reply:
Done